tests/integration_tests_rust.rs

```diff
@@ -910,6 +910,147 @@ fn concurrent_connections_succeed() {
 	}
 }
 
+#[test]
+fn splice_channel() {
+	macro_rules! expect_splice_pending_event {
```

Comment:
nit: Usually we'd add this to `common/mod.rs`.

Reply:
Since it wasn't being used there, having it there would give a build warning.

Reply:
Started looking into this. @tnull The channel funding transaction is recognized as an outbound on-chain payment for A -- presumably because we are spending from the wallet. If B were to splice out paying A (or whomever), should the splice funding transaction be considered an outbound on-chain payment for B? It does not currently since it spends from the funding txo.